about.html

```diff
@@ -0,0 +1,401 @@
+								</li>
+							</ul>
+						</nav>
+					</div>
+				</div>
+			</div>
+		</header>
+
+		<!-- Page content -->
+		<main class="grow">
+
+			<!-- Hero -->
+			<section class="relative">
+
+				<!-- Background image -->
+				<div class="absolute inset-0">
+					<img class="w-full h-full object-cover" src="./images/startup-7376.jpg" width="1440" height="394" alt="About" />
+					<div class="absolute inset-0 bg-zinc-900 opacity-75" aria-hidden="true"></div>
+				</div>
+
+				<!-- Hero content -->
+				<div class="max-w-6xl mx-auto px-4 sm:px-6 relative">
+					<div class="pt-32 pb-12 md:pt-40 md:pb-20">
+						<div class="max-w-3xl mx-auto text-center">
+							<h1 class="h1 mb-4" data-aos="fade-up">The Story of Us</h1>
+							<p class="text-xl text-zinc-400 mb-8" data-aos="fade-up" data-aos-delay="200">We are rebels, makers, and dreamers who got tired of the same shit, different day. We're on a mission to change that.</p>
+						</div>
+					</div>
+				</div>
+
+			</section>
+
+			<!-- Team images -->
+			<section class="relative -mt-8">
+				<div class="max-w-6xl mx-auto px-4 sm:px-6">
+					<div class="max-w-3xl mx-auto">
+						<div class="relative w-full h-0 pb-3/4">
+							<figure class="absolute h-auto" style="top:45%;width: 41.67%;max-width: 320px;" data-aos="fade-right">
+								<img src="./images/htc.jpg" width="320" height="240" alt="Team mosaic 02" />
+							</figure>
+							<figure class="relative mx-auto h-auto" style="width: 78.13%;max-width: 600px;" data-aos="fade-down" data-aos-delay="100">
+								<img src="./images/beiersdorf.png" width="600" height="338" alt="Team mosaic 01" />
+							</figure>
+							<figure class="absolute h-auto" style="top: 8.5%;right: 0;width: 32.55%;max-width: 250px;" data-aos="fade-left" data-aos-delay="200">
+								<img src="./images/Samsung_cover_home.png" width="250" height="188" alt="Team mosaic 03" />
+							</figure>
+							<figure class="absolute h-auto" style="bottom: 0;right: 20%;width: 25.52%;max-width: 196px;" data-aos="fade-up" data-aos-delay="300">
+								<img src="./images/team-mosaic-04.jpg" width="196" height="196" alt="Team mosaic 04" />
+							</figure>
+						</div>
+					</div>
+				</div>
+			</section>
+
+			<!-- Timeline -->
+			<section>
+				<div class="max-w-6xl mx-auto px-4 sm:px-6">
+					<div class="py-12 md:py-20">
+
+						<!-- Section header -->
+						<div class="max-w-3xl mx-auto text-center pb-12 md:pb-20">
+							<h2 class="h2 mb-4">The Backstory</h2>
+							<p class="text-xl text-zinc-400">Who we are is a combination of what we have done, where we are going, and choices we have yet to make.</p>
+						</div>
+
+						<!-- Items -->
+						<div class="max-w-3xl mx-auto -my-4 md:-my-6" data-aos-id-timeline>
```

Comment:
I think either the year bubbles need to be square or the contact buttons also rounded. Perhaps even the company logo frames

Reply:
punt. -> v0.2